fetc/urls.py

```diff
@@ -20,6 +22,16 @@
     path('accounts/login/', auth_views.LoginView.as_view(), name='login'),
     path('accounts/logout/', auth_views.LogoutView.as_view(), name='logout'),
 
+    # Access user uploads
+    path('media/<str:filename>',
+         UserMediaView.as_view(),
+         name='user_upload'),
+    # And a duplicate path for user media
+    # Use this for testing on servers that rewrite /media/
+    path('view_media/<str:filename>',
+         UserMediaView.as_view(),
+         name='user_upload_2'),
```

Comment:
Is this still necessary?

It feels like a fix for uWSGI's static-map, but we are removing that anyways. In addition, we currently don't really control the urls presented to the user, sooooo...